packages/api/src/schema/db-schema.yaml

```diff
@@ -1150,6 +1153,20 @@ components:
         deleted:
           type: boolean
           default: false
+    access-control-key:
```

Comment:
Nit: I think this name is misleading/unclear. WDYT of naming it explicitly like `access-control-blocked-jwt`?